### ISSUE TITLE
API product add resources section scroll is not visible.

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6862]
User does not know its scrollable unless scrolled mistakenly.

![image](https://user-images.githubusercontent.com/20179540/69143452-0379d900-0aef-11ea-9f13-4189dfdb8675.png)
